internal/explorer/models.go

```diff
@@ -139,3 +139,19 @@ type farmData struct {
 type FarmResult struct {
 	Data farmData `json:"data"`
 }
+
+type countersReturnValue struct {
+	NodeCapacity gridtypes.Capacity `json:"nodeCapacity"`
+	UsedCapacity gridtypes.Capacity `json:"usedCapacity"`
+	Err          error              `json:"err"`
```

Comment:
If these are not (un)marshalled, the json annotation is redundant/confusing (for me).